source/FreeRTOS_IP.c

```diff
@@ -1562,7 +1562,7 @@ static eFrameProcessingResult_t prvAllowIPPacket( const IPPacket_t * const pxIPP
                                         /* Exclude this from branch coverage as this is only used for debugging. */
                                         if( xCount < 5 ) /* LCOV_EXCL_BR_LINE */
                                         {
-                                            FreeRTOS_printf( ( "prvAllowIPPacket: UDP packet from %xip without CRC dropped\n",
+                                            FreeRTOS_printf( ( "prvAllowIPPacket: UDP packet from %lxip without CRC dropped\n",
```

Comment:
Hi @htibosch ,

I have been comparing my local repository with the on-line repository and somehow it appears to be out of date on my local hard drive and I can not explain why!  I am going to close my two recent PR's, delete my local repositories and clone them again.  I will then rebuild my projects and see what is still not correct and resubmit new pull requests at that point.  I am very sorry to waste time!

Kind Regards,

Pete

Reply:
> somehow it appears to be out of date on my local hard drive

It may help to open your github page and press a button called "sync fork".

Note that there are 3 copies of the library:
1) On github/FreeRTOS
2) On github/pete-pjb
3) On your harddisk

"sync fork" will update the 2nd copy. Your github client will update the third copy.

( git is a genius product but not always easy to understand )

> I am going to close my two recent PR's, delete my local repositories and clone them again

That's the way to go, start from scratch.

> I am very sorry to waste time!

You are not wasting anyone's time, our discussion may be useful for other developers.

Reply:
Hi @htibosch ,

Thank you for your reply,  I have now sent you an email at your email address hein_tibosch <at> yahoo <dot> es with a bit more information.

Kind Regards,

Pete